lib/l10n/app_en.arb

```diff
@@ -193,10 +193,93 @@
-  "pageHelpWhereIsAccessibilityQuestion": "Where are the accessibility settings?",
-  "pageHelpWhereIsAccessibilityAnswer": "This app supports a multitude of accessibility settings offered by the OS. Go to your OS's settings to enable Screen Reader or large fonts!",
+  "pageHelpEmailSubject": "Inquiry by User-ID [{id}]",
+  "@pageHelpEmailSubject": {
+    "placeholders": {
+      "id": {}
+    }
+  },
+  "pageHelpQuestion1": "What is MOTIS?",
+  "pageHelpAnswer1": "MOTIS encompasses many tools to support intermodal routing. Its goal is to help plan journeys that include public transport, bike sharing, ride sharing and more modes of transportation.",
+  "pageHelpQuestion2": "How much do rides cost? How can I pay?",
+  "pageHelpAnswer2": "We designed the payment process to fully rely on user agreements. Both the cost and payment method of a ride can be negotiated in person or in our in-app chat. You will see a price suggestion for most rides as a starting point.",
+  "pageHelpQuestion3": "We agreed on a price, but now my rider isn't paying it. What can I do?",
+  "pageHelpAnswer3": "We're sorry for the inconvenience. Unfortunately, the only thing we can offer you is the ability to report users in that case. Tap the \"{reportButtonText}\" button on their profile and make sure to select the reason \"{didNotPayReasonText}\".",
+  "@pageHelpAnswer3": {
+    "placeholders": {
+      "reportButtonText": {},
+      "didNotPayReasonText": {}
+    }
+  },
+  "pageHelpQuestion4": "A driver/rider didn't show up. What can I do?",
+  "pageHelpAnswer4": "We're sorry for the inconvenience. Unfortunately, the only thing we can offer you is the ability to report users in that case. Tap the \"{reportButtonText}\" button on their profile and make sure to select the reason \"{didNotShowUpReasonText}\".",
+  "@pageHelpAnswer4": {
+    "placeholders": {
+      "reportButtonText": {},
+      "didNotShowUpReasonText": {}
+    }
+  },
+  "pageHelpQuestion5": "My ride got cancelled. What can I do?",
+  "pageHelpAnswer5": "We're sorry for the inconvenience. Unfortunately, you will have to look for another ride. If the driver cancelled the drive just before the trip, you can report them using the \"{reportButtonText}\" button on their profile. Make sure to select the reason \"{otherReasonText}\" and explain the situation.",
+  "@pageHelpAnswer5": {
+    "placeholders": {
+      "reportButtonText": {},
+      "otherReasonText": {}
+    }
+  },
+  "pageHelpQuestion6": "My ride was rejected. What can I do?",
+  "pageHelpAnswer6": "We're sorry for the inconvenience. Unfortunately, you will have to look for another ride.",
+  "pageHelpQuestion7": "The driver isn't reacting to my request. What can I do?",
+  "pageHelpAnswer7": "We're sorry for the inconvenience. If you tap on the drive, you can take the request back using the \"{withdrawButtonText}\" button at the bottom. Then, you can look for a new ride.",
```

Comment:
can we change this. It makes the user think that only one ride can be requested at the same time. But if I'm correct thats possible. Maybe just something like: 
"We're sorry for the inconvenience. The only thing you can do is look for a new ride. You can also tap on the drive, there you can take the request back using the \"{withdrawButtonText}\" button at the bottom. "

Reply:
Hmm the idea was that we don't want to endorse ride request spamming where a user would request all possible rides for him and then choose.
We are not sure what the best combination of these two ideas would be.

Reply:
We changed the text, what do you think?

Reply:
yes look's good